### ISSUE TITLE
The plugin uses a deprecated version of the Android embedding

### ISSUE THREAD

**Issue by miguelpruivo** [builttoroam/device_calendar#350]
To avoid unexpected runtime failures, or future build failures, try to see if this plugin supports the Android V2 embedding. Otherwise, consider removing it since a future release of Flutter will remove these deprecated APIs.
If you are plugin author, take a look at the docs for migrating the plugin to the V2 embedding: https://flutter.dev/go/android-plugin-migration.

**Comment by thomassth** [comment]
Duplicate of #326. Fixed in #327.

Please try to search for issues (both open and closed) before starting your own. I found it just by typing "deprecated".

**Comment by miguelpruivo** [comment]
Sorry, quick searched but didn't find actually.

**Comment by thomassth** [comment]
The result usually only show open issues by default. You need to consciously choose to see closed ones as well.

We'll probably do a release soon so hopefully there will be no more issues like this :)

**Comment by Zookey** [comment]
@thomassth Is this released? Is it safe to use this library without this fix in production?

**Comment by thomassth** [comment]
Our source code is always released ;)

But seriously, we are about to push new releases to pub.dev right after we fix the CI/CD. Meanwhile you can grab the dev branch and sideload it.

**Comment by Zookey** [comment]
Thanks. Is it possible to publish without CI/CD, since this is breaking changes and device_calendar will not work for the apps in production that are using Flutter 2.5.3.

@thomassth

**Comment by thomassth** [comment]
Credentials are needed to publish, which I don't.

See #346 for related updates.

For sideloading plugins see [here](https://docs.flutter.dev/development/packages-and-plugins/using-packages#dependencies-on-unpublished-packages)
